install_config/configuring_aws.adoc

```diff
@@ -310,18 +310,26 @@ There are four types of resources that need to be tagged:
 [[aws-tagging-an-existing-cluster]]
 === Tagging an Existing Cluster
 
-A cluster uses the value of the `kubernetes.io/cluster/<name>,Value=<clusterid>` tag to determine which
-resources belong to the cluster. Therefore, you must tag all resources with the
-key `kubernetes.io/cluster/<name>,Value=<clusterid>` and have the same value for that key.
+A cluster uses the value of the `kubernetes.io/cluster/<name>,Value=<clusterid>`
+tag to determine which resources belong to the AWS cluster. Therefore, tag all
+relevant resources with the `kubernetes.io/cluster/<name>,Value=<clusterid>`
+key, and have the same value for that key.
```

Comment:
I think the way that we're referring to the key is a little confusing. 
Can you add a note here that explains the `name` and `clusterid` parameter values? It sounds like `clusterid` needs to be some constant, arbitrary value, but IDK what the `name` should be.